yarn.lock

```diff
@@ -7875,6 +7875,16 @@ react-native-macos@^0.63.33:
     use-subscription "^1.0.0"
     whatwg-fetch "^3.0.0"
 
+  version "0.6.10"
+  resolved "https://registry.npmjs.org/react-native-test-app/-/react-native-test-app-0.6.10.tgz#426670261c43d17a4a1e74419cba6f5691dce88f"
+  integrity sha512-QnG+A9oaf20qRCsmnWh2WYfebmu/kn8DGN3OAUaNRpLaIh6j9/t5HIaT+6cn5bEmNpELCPaN1hfgyl2KmtE3RA==
+  dependencies:
+    chalk "^4.1.0"
+    prompts "^2.4.0"
+    rimraf "^3.0.0"
+    semver "^7.3.5"
+    yargs "^16.0.0"
+
```

Comment:
Can you please revert this file, otherwise `yarn` will complain about an invalid lock file.

Reply:
Sorry, I meant can we undo all changes within this file. This section was added because you added `react-native-test-app` to `package.json` earlier.